src/Tickets/Commerce/Currency.php

```diff
@@ -11,4 +11,32 @@
  */
 class Currency {
 
+	/**
+	 * Retrieves the working currency set in Tribe Commerce or Tickets Commerce
+	 *
+	 * @todo @backend: due to time constraints, not all calls to tribe_get_option( 'ticket-commerce-currency-code', 'USD' )
+	 *       were replaced by this method. We need to cleanup and have the entire codebase
+	 *       read currency codes from this class in the future. Once all currency operations are fixed to use this method,
+	 *       modify this method to delete the old keys from the settings array.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+
+	public static function get_currency() {
```

Comment:
We might want to be more specific and call this `get_currency_code` so there's no confusion with things like 'currency symbol' or whatever else might come up in the future.

Reply:
good thinking